llpc/lower/llpcSpirvLower.cpp

```diff
@@ -154,6 +154,24 @@ void SpirvLower::removeConstantExpr(Context *context, GlobalVariable *global) {
     replaceConstWithInsts(context, constVal);
 }
 
+// =====================================================================================================================
+// Replace global variable with another global variable
+//
+// @param global : Replaced global variable
+// @param replaceGlobalglobal : Replacing global variable
+void SpirvLower::replaceGlobal(GlobalVariable *global, GlobalVariable *replaceGlobal) {
```

Comment:
nit: the first parameter has a very generic name (`global`). Consider calling these  something like `oldGlobal`, `newGlobal`, or `original`, `replacement`.